contracts/registry/RegistryService.sol

```diff
@@ -201,21 +167,13 @@ contract RegistryService is
     }
 
     // From IService
-    function getName() public pure override(IService, Service) returns(string memory) {
-        return NAME;
+    function getType() public pure override(IService, Service) returns(ObjectType serviceType) {
+        return SERVICE(); 
```

Comment:
I don't like this naming. The RegistryService is `SERVICE` and then other services are `PRODUCT`, `DISTRIBUTION` and `POOL`. It does not feel consistent to me. Maybe we should not use the ObjectType here or at least create dedicated object types `REGISTRY_SERVICE`, `PRODUCT_SERVICE`, ...

Reply:
Revert the type back to `SERVICE` (for all ... not only for registryService) and add a function `getDomain` which returns the _domain_ of the service (`POOL`, `PRODUCT`, ...)

Reply:
getType() is applicable only to IService now thus propose just rename it

Reply:
done